docs/sources/alerting/fundamentals/alert-rules/alert-rule-types.md

```diff
@@ -20,7 +20,7 @@ In addition to supporting multiple data sources, you can also add expressions to
 
 The following diagram shows how Grafana-managed alerting works.
 
-{{< figure src="/media/docs/alerting/grafana-managed-rule.png" max-width="750px" caption="How Alerting works" >}}
+{{< figure src="/media/docs/alerting/grafana-managed-rule.png" max-width="750px" caption="Grafana-managed alerting" >}}
```

Comment:
The hyphen is fine afaik because it is Grafana Labs writing it as opposed to someone else. Leaving as-is.